src/cli_shared/cli/mod.rs

```diff
@@ -244,8 +238,8 @@ impl CliOpts {
         // (where to find these flags, should be easy to do with structops)
 
         // check and set syncing configurations
-        if let Some(req_window) = &self.req_window {
-            cfg.sync.req_window = req_window.to_owned();
+        if let Some(req_window) = self.req_window {
+            cfg.sync.request_window = req_window;
```

Comment:
`cfg.sync.request_window` was a duplicate of `ChainConfig::request_window` and never used.